examples/3d/msaa_pipelined.rs

```diff
@@ -0,0 +1,65 @@
+use bevy::{
+    ecs::prelude::*,
+    input::Input,
+    math::Vec3,
+    pbr2::{PbrBundle, PointLightBundle, StandardMaterial},
+    prelude::{App, Assets, KeyCode, Transform},
+    render2::{
+        camera::PerspectiveCameraBundle,
+        color::Color,
+        mesh::{shape, Mesh},
+        view::Msaa,
+    },
+    PipelinedDefaultPlugins,
+};
+
+/// This example shows how to configure Multi-Sample Anti-Aliasing. Setting the sample count higher
+/// will result in smoother edges, but it will also increase the cost to render those edges. The
+/// range should generally be somewhere between 1 (no multi sampling, but cheap) to 8 (crisp but
+/// expensive)
+fn main() {
+    println!("Press 'm' to toggle MSAA");
+    println!("Using 4x MSAA");
+    App::new()
+        .insert_resource(Msaa { samples: 4 })
+        .add_plugins(PipelinedDefaultPlugins)
+        .add_startup_system(setup.system())
+        .add_system(cycle_msaa.system())
```

Comment:
```suggestion
        .add_startup_system(setup)
        .add_system(cycle_msaa)
```
`.system()` is now optional